mtz-url-rewrite-interceptor.html

```diff
@@ -111,7 +111,7 @@
         isLocal = this.__isLocal, localBaseUrl = this.localBaseUrl, baseUrl = this.baseUrl,
         pattern = this.pattern, replacer = this.replacer, replacement = this.replacement, origin = this.__origin) {
         // Determine baseUrl
-        baseUrl = isLocal ? localBaseUrl : baseUrl || origin;
+        baseUrl = isLocal && localBaseUrl ? localBaseUrl : baseUrl || origin;
```

Comment:
You need to add `()` to scope this and ensure it's running in the correct order.  The op order in this case is going to be `&&` > `||` > `... ? ... : ...` which equates to:
`baseUrl = `(isLocal && localBaseUrl) ? localBaseUrl : (baseUrl || origin);` which doesn't seem right to me.

If you're curious about order
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence

Reply:
Nope that's correct.

Reply:
I may go ahead and add them just for clarity but it should resolve correctly.